### ISSUE TITLE
Mounts & Pets Management

### ISSUE THREAD

**Issue by Kalima** [fvtt-fria-ligan/forbidden-lands-foundry-vtt#59]
Currently mounts are listed as Gear, and lack the ability to have items attached to them, as well as food, water and consumables. Adding an animal as an item doesn't make much sense.

One current workaround is to use a player character sheet, but it doesn't really function properly. To give an example, if a goblin uses a boar to carry his gear, the boar shouldn't be able to use weapons stored on his back, but it should be able to use its tusks, which is its natural weapon. Currently, any weapon stored on a mount appears in the Attacks tab; same behavior with armors. Furthermore a group of players can share a mount as a beast of burden, and having it attached to one particular character can make this cumbersome.

For these reasons, I think it would be preferable if mounts had their own character sheet.  Pets are almost always attached to a specific character, but would also benefit from having their own sheet.

An option would be to make it possible to drag and drop a mount or a pet character sheet in an "Extras" tab on the players' characters' sheets, to link them without having to treat mounts and pets as items (which makes little sense mechanically since they have different attributes and features than pure items).

To summarize, an ideal mounts/pets' sheet could include the following:
- Main tab with attributes/skills;
- Combat tab with natural attacks, armor (and possibly weapons) and critical injuries;
- Gear tab with mount gear, consumables, and a separate section for cargo;
- Notes tab;
- Useful but not needed, a section displaying the links created between the player characters and the mount/pet, to visualize who has a relationship with and can access the sheet. That section would be automatically populated when the mount/pet is dragged and dropped to a character sheet.
